circuit_maintenance_parser/parsers/turkcell.py

```diff
@@ -0,0 +1,69 @@
+"""Turkcell parser."""
+import logging
+import re
+from typing import Dict
+
+from bs4.element import ResultSet  # type: ignore
+from dateutil import parser
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-nested-blocks, too-many-branches
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserTurkcell1(Html):
+    """Notifications Parser for Turkcell notifications."""
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_tables(soup.find_all("table"), data)
+            return [data]
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_tables(self, tables: ResultSet, data: Dict):
+        """Parse tables."""
```

Comment:
I think as a general practice (even if we haven't been doing this previously) it would be a **really helpful thing** if in the docstring or a comment we were to include a stripped-down example of the HTML format we're expecting to find and parse, something like:

```suggestion
        """Parse tables.
        
        Format 1 supported here (all information in a single table):
            <table>
                <tr>
                    <td>Dear Customer ... planned ...</td>
                    <td>...</td>
                </tr><tr>
                    <td>Maintenance Number</td>
                    <td>123456789</td>
                </tr><tr>
                    <td>Start</td>
                    <td>2021-08-31 01:02:03</td>
                    ...
                    
        Format #2 supported here (one table for general information, a separate table listing impacted circuits):
            ...
        """
```

Reply:
Yes, I agree that this would be really useful to provide context for future reviews.